tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -362,6 +363,19 @@ func (az *AzureProvider) createSecurityGroupRules(ctx context.Context, nsgName s
 							Priority:                 to.Int32Ptr(200),
 						},
 					},
+					{
+						Name: to.StringPtr("WINRM"),
+						SecurityRulePropertiesFormat: &network.SecurityRulePropertiesFormat{
+							Protocol:                 network.SecurityRuleProtocolTCP,
+							SourceAddressPrefix:      to.StringPtr(*myIP + "/32"),
+							SourcePortRange:          to.StringPtr("*"),
+							DestinationAddressPrefix: to.StringPtr("*"),
+							DestinationPortRange:     to.StringPtr("5986"),
+							Access:                   network.SecurityRuleAccessAllow,
+							Direction:                network.SecurityRuleDirectionInbound,
+							Priority:                 to.Int32Ptr(300),
```

Comment:
Why 300 as priority?

Reply:
Please add a comment saying why you picked 300. I am guessing this can be any number greater than 200.